test/data/bittorrent-dht-client.js

```diff
@@ -143,7 +143,7 @@ function DHT(opts) {
 
```

Comment:
You should never need to `return undefined` in a function since `return` implicitly returns undefined.